api/v1alpha1/replicationsource_types.go

```diff
@@ -111,13 +111,19 @@ type ReplicationSourceRsyncSpec struct {
 	SSHUser *string `json:"sshUser,omitempty"`
 }
 
+// ReplicationSourceRcloneSpec defines the field for rclone in replicationSource.
 type ReplicationSourceRcloneSpec struct {
 	ReplicationSourceVolumeOptions `json:",inline"`
-	// rcloneConfig is the name of a Secret that contains a valid Rclone
-	// configuration file.
-	RcloneConfig string `json:"rcloneConfig,omitempty"`
-	// path is the remote path to sync to.
-	Path string `json:"path,omitempty"`
+	//RcloneConfigSection is the section in rclone_config file to use for the current job.
+	RcloneConfigSection *string `json:"rcloneConfigSection,omitempty"`
```

Comment:
Can this and the next 5 fields be treated as an anonymous type included in the struct here and above?

Reply:
As long as we can't think of a reason why they would diverge in the future, it seems reasonable.

Reply:
Right. They _could_ diverge at some point. I was thinking that the fields common to both source and dest could be defined in a struct and then embedded into the larger source and dest structures. Of course, both the larger source and dest structs could contain separate field unique to a source or destination operation.